hyp3_autorift/process.py

```diff
@@ -63,9 +63,14 @@ def get_lc2_metadata(scene_name):
 
 
 def get_lc2_path(metadata):
```

Comment:
Are there tests for this function that need to be updated? If not, should we add some?